pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -350,7 +351,7 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication, ReentrancyGuard {
         address account,
         address where
     ) public view returns (bool) {
-        return _hasPermissionSpecificallyOrGenerally(GRANT_ACTION_ID, account, where, actionId);
+        return _isGranter[actionId][where][account] || _isGranter[actionId][EVERYWHERE][account] || isRoot(account);
```

Comment:
`_isGranter` is defined as `action id => account => where => is granter`.

Shouldn't this be `_isGranter[actionId][account][where] || _isGranter[actionId][account][EVERYWHERE]`?